### PR TITLE
Add options in iotlab-auth for setting an SSH key

### PR DESCRIPTION
**Pull request by aabadie**
This PR is a rework from #5.

closes #5

### CODE REVIEW

File: iotlabcli/auth.py [hunk 1]
```diff
@@ -85,3 +85,27 @@ def _read_password_file():
             return username, password
     except ValueError:
         raise ValueError('Bad password file format: %r' % RC_FILE)
+
+
+IDENTITY_FILE = os.path.expanduser('~/.ssh/id_rsa')
+
+
+def add_ssh_key(identity_file=None):
+    """Install ssh key into user's iot-lab account on www.iot-lab.info"""
```
Comment (fsaintma) [inline iotlabcli/auth.py:94]: In this comment I suggest you to remove www.iot-lab.info. (it could be DEV)

**Comment by aabadie** [comment]
> Indeed it would be great to print ssh keys which has been already uploaded ?

Done, just need to add the tests  now :)

**Comment by aabadie** [comment]
@fsaintma, I added the `-l/--list-ssh-keys` options.

File: iotlabcli/auth.py [hunk 2]
```diff
@@ -85,3 +87,39 @@ def _read_password_file():
...
+
+
+def ssh_keys():
+    """List ssh keys configured into user's iot-lab account"""
+
+    api = Api(*get_user_credentials())
+    keys_json = api.get_ssh_keys()
+
+    print("SSH keys:")
+    for key in keys_json['sshkeys']:
```
Comment (fsaintma) [inline iotlabcli/auth.py:124]: Why don't you return the JSON like the other commands ?

Comment (aabadie) [reply iotlabcli/auth.py:124]: The other values returned by iotlab-auth don't return values as JSON, just plain text. I couldn't find a use case with SSH keys.

Comment (fsaintma) [reply iotlabcli/auth.py:124]: ok I merge your PR

**Pull request merged**